packages/plans-grid/src/plans-feature-list/style.scss

```diff
@@ -22,123 +30,159 @@ ul.plans-feature-list__item-group {
+
+			&:hover,
+			&:focus {
+				color: var( --studio-orange-30 );
+			}
 		}
 	}
+}
 
-	.plans-feature-list__item-url {
-		word-break: break-all;
-	}
+// SVG used as the bullet icon
+.plans-feature-list__item-bullet-icon {
+	flex-grow: 0;
+	flex-shrink: 0;
+	margin-right: 6px;
 
-	// the tick
-	svg:first-child {
-		// we use a margin because there is a space,
-		// the strike-through will look dirty over it
-		margin-right: 5px;
-		vertical-align: middle;
-	}
+	color: var( --studio-gray-100 );
 
-	svg:first-child path {
-		fill: var( --studio-green-40 );
+	.plans-accordion & {
+		color: var( --studio-green-60 );
+	}
 
-		// to make the tick thicker
-		stroke: var( --studio-green-40 );
+	.plans-feature-list__item-content-wrapper--domain-button.is-unavailable &,
+	.plans-feature-list__item--disabled-message &,
+	.plans-feature-list__item--requires-annual-disabled & {
+		color: var( --studio-orange-40 );
 	}
 
-	&.is-picked {
-		font-weight: bold;
+	&,
+	path {
+		fill: currentColor;
+		stroke: currentColor;
 	}
+}
 
-	&.is-cta {
-		font-weight: bold;
-		padding: 0;
+// Wrapper for the feature item's text content
+.plans-feature-list__item-text {
+	flex-grow: 1;
+	flex-shrink: 0;
+	max-width: 100%;
 
-		// overwrite button styles
-		&.components-button.is-link {
-			color: var( --studio-blue-40 );
-		}
+	// Makes the tick/cross icon align better with the text
+	margin-bottom: 1px;
+}
 
-		// the tick
-		svg:first-child path {
-			fill: var( --studio-green-40 );
-			stroke: var( --studio-green-40 );
-			margin-top: 5px;
-		}
+// Optional text informing the user that a specific feature is only available with the annual version of a plan
+.plans-feature-list__item-annual-nudge {
+	display: block;
+	text-transform: uppercase;
+	font-size: 10px; /* stylelint-disable-line scales/font-size */
+	font-weight: 700;
+	letter-spacing: 0.02em;
+	line-height: 1;
+	margin-bottom: 6px;
+
+	.plans-feature-list__item--requires-annual-enabled & {
+		color: var( --studio-green-60 );
+	}
 
-		// the chevron
-		svg:last-child {
-			fill: var( --studio-blue-40 );
-			stroke: var( --studio-blue-40 );
-			margin-left: 8px;
-		}
+	.plans-feature-list__item--requires-annual-disabled & {
+		color: var( --studio-orange-40 );
 	}
 }
 
+// The text describing the feature
+.plans-feature-list__item-description {
+	.plans-feature-list__item-content-wrapper--domain-button.is-cta & {
+		text-decoration: underline;
+	}
+
+	.plans-feature-list__item--disabled-message &,
+	.plans-feature-list__item--disabled-message
+		.plans-feature-list__item-content-wrapper--domain-button.is-cta
+		&,
+	.plans-feature-list__item--requires-annual-disabled &,
+	.plans-feature-list__item--requires-annual-disabled
+		.plans-feature-list__item-content-wrapper--domain-button.is-cta
+		& {
+		text-decoration: line-through;
+	}
+}
+
+.plans-feature-list__item-url {
+	word-break: break-all;
+}
+
 .plans-feature-list__placeholder {
 	@include onboarding-placeholder();
 	display: inline-block;
```

Comment:
We need more spacing between column on feature list. The overlapping is visible on languages with longer texts or on smaller screen sizes.
Probably related to the changes in this file.

<img width="802" alt="Screenshot 2021-01-27 at 12 40 56" src="https://user-images.githubusercontent.com/14192054/105980785-f61a9000-609d-11eb-8122-4575ed3b8560.png">

Reply:
Good catch! Should be fixed in https://github.com/Automattic/wp-calypso/pull/48963/commits/ba30e28e653c7227085cdf0d32ae4b9b6b441a72 

With verbose languages, the feature layout doesn't always look pretty, but I don't think it should be a blocker for this PR
<img width="967" alt="Screenshot 2021-01-27 at 13 14 44" src="https://user-images.githubusercontent.com/1083581/105989584-a0e07d80-60a1-11eb-9066-443290e7c347.png">